Sources/Components/Components/Watchlist/OldWKWatchlistViewController.swift

```diff
@@ -0,0 +1,198 @@
+import Foundation
```

Comment:
Feel free to delete these old classes if you'd like, I don't need them and you can always pull up an old PR if for whatever reason you wanted to reference them.

Reply:
I'll rip them out in my next stacked PR.